### PR TITLE
docs(changelog): release notes for v0.18.0

### PR DESCRIPTION
**Pull request by vpavlin**
# Description
This PR updates CHANGELOG for release [v0.18.0](https://github.com/waku-org/nwaku/milestone/10)

Please check if the changelog looks ok and propose highlights!

### CODE REVIEW

**Review by Ivansete-status — APPROVED** [review]
LGTM, thanks for that!

Shall we mention that this is the last version supporting `--topics` parameter?

In the `Upgrade instructions`:
The `--topics` config option has been deprecated to unify the configuration style. It is still available in this next release but will be removed afterward. The new option `--topic` is introduced, which can be used repeatedly to achieve the same behavior.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
+## 2023-06-14 v0.18.0
+
+> Note that there is a new naming scheme for release artifacts.
+
+## What's Changed
+
+Release highlights:
```
Comment (rymnc) [inline CHANGELOG.md:7]: Can we include that rln-relay uses atomic operations now? This is a milestone :)

**Review by rymnc — APPROVED** [review]
LGTM, just a single comment :)

**Review by jm-clius — APPROVED** [review]
Thanks! LGTM.

**Pull request merged**